shared/src/business/utilities/pdfGenerator/documentTemplates/PrintableWorkingCopySessionList.jsx

```diff
@@ -0,0 +1,206 @@
+const React = require('react');
+import { PrimaryHeader } from '../components/PrimaryHeader';
+import { ReportsHeader } from '../components/ReportsHeader';
+import { SessionNotesSection } from '../components/SessionNotesSection';
+
+const isMemberCase = formattedCase => {
+  return formattedCase.inConsolidatedGroup && !formattedCase.leadCase;
+};
+
+const orderedFilterMap = [
+  {
+    code: 'setForTrial',
+    name: 'Set For Trial',
+  },
+  {
+    code: 'dismissed',
+    name: 'Dismissed',
+  },
+  {
+    code: 'continued',
+    name: 'Continued',
+  },
+  {
+    code: 'rule122',
+    name: 'Rule 122',
+  },
+  {
+    code: 'aBasisReached',
+    name: 'A Basis Reached',
+  },
+  {
+    code: 'settled',
+    name: 'Settled',
+  },
+  {
+    code: 'recall',
+    name: 'Recall',
+  },
+  {
+    code: 'takenUnderAdvisement',
+    name: 'Taken Under Advisement',
+  },
+  {
+    code: 'statusUnassigned',
+    name: 'Status Unassigned',
+  },
+];
+
+const generateSelectedFilterList = filters => {
+  const selectedFilters = [];
+  const filterKeys = Object.keys(filters).filter(
+    filterKey => filterKey !== 'showAll',
+  );
+  const userFilterSelection = filterKeys.filter(
+    filterKey => filters[filterKey],
+  );
+
+  orderedFilterMap.map(orderedFilterKey => {
```

Comment:
Nit: my `deepscan` plugin actually pointed this out - since nothing is being used from the return of this `map`, it could be a `forEach`

Reply:
Done